Backtesting/collectors/exchanges/exchange_history_collector/tests/test_history_collector.py

```diff
@@ -86,4 +89,57 @@ async def test_collect_invalid_data():
         assert collector.exchange_manager is None
         assert collector.exchange is not None
         assert collector.file_path is not None
+        assert collector.temp_file_path is not None
+        assert not os.path.isfile(collector.temp_file_path)
+
+async def test_collect_valid_date_range():
+    exchange_name = "binance"
+    tentacles_setup_config = test_utils_config.load_test_tentacles_config()
+    symbols = ["ETH/BTC"]
+    async with data_collector(exchange_name, tentacles_setup_config, symbols, None, True, 1549065660000, 1549670520000) as collector:
```

Comment:
We need to update `async def data_collector()` to take those 2 last arguments into account :)

Reply:
ah yes !